packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -570,7 +576,9 @@ export class PipelineEditor extends React.Component<
   }
 
   handleOpen(): void {
+    console.log('>>> handleOpen');
     toArray(this.browserFactory.defaultBrowser.selectedItems()).map(item => {
+      console.log('Opening ==> ' + item.path);
```

Comment:
This logging seems redundant.  I don't think either of these is necessary, especially when there's already the same log statement happening below if the item is not a directory.